.jenkins/caffe2/test.sh

```diff
@@ -130,7 +130,10 @@ if [[ "$BUILD_ENVIRONMENT" == *onnx* ]]; then
   # JIT C++ extensions require ninja, so put it into PATH.
   export PATH="/var/lib/jenkins/.local/bin:$PATH"
   if [[ "$BUILD_ENVIRONMENT" == *py3* ]]; then
-    pip install -q --user onnxruntime==0.4.0
+    # default pip version is too old(9.0.2), unable to support tag `manylinux2010`.
+    # Fix the pip error: Couldn't find a version that satisfies the requirement
+    sudo pip install pip==19.2
```

Comment:
could you try doing `pip install -q --user --update pip setuptools` (i.e. no sudo, and not hard code pip version, and also update setuptools)

Reply:
it seems sudo is needed to upgrade the pip that is being used here ... 

> Aug 20 02:06:28 + pip install -q --user --upgrade pip setuptools
Aug 20 02:06:31 + pip install -q --user -i https://test.pypi.org/simple/ ort-nightly==0.5.0.dev817
Aug 20 02:06:32   Could not find a version that satisfies the requirement ort-nightly==0.5.0.dev817 (from versions: 0.3.1.dev414, 0.3.1.dev419, 0.3.1.dev423, 0.3.1.dev425, 0.4.0.dev428, 0.4.0.dev507)
Aug 20 02:06:32 No matching distribution found for ort-nightly==0.5.0.dev817

Reply:
I see, yeah use sudo then (just noticed there are other places in this script using sudo already)

Reply:
CI passing now, failure seem unrelated